binder/environment.yml

```diff
@@ -1,45 +1,18 @@
 channels:
-- conda-forge # Used by jupyter-desktop-server
+# install a nightly build of napari
+  - napari/label/nightly
+  - conda-forge # Used by jupyter-desktop-server
 dependencies:
-# See: https://github.com/conda-forge/napari-feedstock/blob/master/recipe/meta.yaml
-- python >=3.8
-# dependencies matched to pip
-- appdirs >=1.4.4
-- cachey >=0.2.1
-- certifi >=2020.6.20
-- dask >=2.1.0
-- imageio >=2.20
-- importlib-metadata >=1.5.0  # not needed for py>37 but keeping for noarch
-- jsonschema >=3.2.0
-- magicgui >=0.7.0
-- napari-console >=0.0.4
-- napari-plugin-engine >=0.1.9
-- napari-svg >=0.1.4
-- numpy >=1.18.5
-- numpydoc >=0.9.2
-- pillow
-- pint >=0.17
-- psutil >=5.0
-- pyopengl >=3.1.0
-- pyyaml >=5.1
-- pydantic >=1.8.1
-- qtpy >=1.7.0
-- scipy >=1.2.0
-- superqt >=0.4.1
-- tifffile >=2020.2.16
-- typing_extensions
-- toolz >=0.10.0
-- tqdm >=4.56.0
-- vispy >=0.6.4
-- wrapt >=1.11.1
+  - python = 3.12
+  - napari
```

Comment:
With this change, we get binder installing the released version instead of development one. Is it intentional?

Reply:
It's the nightly. I think that's a reasonable compromise for the simplicity and much faster build?